components/Modules/module-detail-page.tsx

```
@@ -798,19 +800,19 @@ export default function ModuleDetailWrapper({
               </div>
             ) : (
               <p
-                className="text-muted-foreground cursor-pointer hover:bg-muted/50 p-4 rounded min-h-[158px]"
+                className="text-muted-foreground cursor-pointer hover:bg-muted/50 p-4 mr-4 rounded min-h-[158px]"
                 onClick={() => setIsEditingDescription(true)}
               >
-                {module.description || "Give context to your agent..."}
+                {module.description || "Provide context to your agent..."}
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)